Makefile

```diff
@@ -1,4 +1,4 @@
-STACK_VERSION ?= v0.2.1
+STACK_VERSION ?= local
```

Comment:
Looked into this just to verify (and because I haven't looked into these shared CI pipelines before) and this appears to be good :+1: For anyone else who is reviewing you can see how tagging happens here: https://github.com/crossplane/cicd/blob/b39e4fb77434c70838474e983ccdb009787ff848/vars/runStackContinuousPipeline.groovy#L59